postmark/core.py

```diff
@@ -338,9 +345,13 @@ def _check_values(self):
             raise PMMailMissingValueException('Cannot send an e-mail without a sender (.sender field)')
         elif not self.__to:
             raise PMMailMissingValueException('Cannot send an e-mail without at least one recipient (.to field)')
-        elif not self.__subject:
+        elif bool(self.__template_id) != bool(self.__template_model):
```

Comment:
Opps. I changed that code when I added tests. Do I need to do a new pull request?